boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts

```diff
@@ -83,13 +83,18 @@
 		regulator-boot-on;
 	};
 
-	nrf_radio_fem: fem {
```

Comment:
What happened to the `fem` node? I'm afraid with this patch nRF21540 in Thingy53 might be not working anymore.

Reply:
Disregard. I see this one was a duplicate of the `fem` node from thingy53_nrf5340_cpunet.dts